projects/Mallard/src/screens/issue-screen.tsx

```diff
@@ -136,8 +140,23 @@ const IssueScreenWithPath = ({ path }: { path: PathToIssue | undefined }) => {
                         </FlexCenter>
                     </>
                 ),
-                success: issue => (
+                success: (issue, { retry }) => (
                     <>
+                        {preview && (
+                            <Button
+                                onPress={() => {
+                                    clearCache()
+                                    retry()
+                                }}
+                                buttonStyles={{
+                                    borderRadius: 0,
+                                    backgroundColor:
```

Comment:
not a fan of injecting styles into the button because this makes changing the component down the line super brittle, it means we gotta check that any changes work with any extra styles added. Could we use the existing button appearance prop to make pink buttons?